server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -760,6 +764,15 @@ private void findStorageGroup(MNode node, String path, HashSet<String> res) {
     }
   }
 
+  /**
+   * Get all devices in current Metadata Tree.
+   *
+   * @return a list contains all distinct storage groups
+   */
+  HashSet<String> getAllDevices() {
+    return new HashSet<>(getNodesList("3"));
```

Comment:
why not using an int?

Reply:
`getNodesList` method is implemented before and I just call it here, should I change the param type to int? (which I think is better as well)